contrib/windows/build-installer.nsi

```diff
@@ -29,7 +29,7 @@ FunctionEnd
 Function createDesktopLink
   ${NSD_GetState} $Checkbox $0
   ${If} $0 <> 0
-    CreateShortCut "$DESKTOP\julia.lnk" "$INSTDIR\bin\julia.exe"
+    CreateShortcut "$DESKTOP\julia.lnk" "$INSTDIR\mintty\usr\bin\env.exe" "HOME=%HOMEDRIVE%%HOMEPATH% $INSTDIR\mintty\usr\bin\mintty.exe -h error -i $INSTDIR\bin\julia.exe $INSTDIR\bin\julia.exe" "$INSTDIR\bin\julia.exe" 0 SW_SHOWNORMAL "" "The Julia REPL"
```

Comment:
I included the `-h error` on the `mintty` command line to hold the window open when `julia` exits with something other than 0. Personally, I think this is a good feature to keep in indefinitely, but it should be especially useful in helping people make bug reports as we try out this `mintty` thing.

The one downside: it makes the already long shortcut command line that much longer. People installing julia deep in a tree (like `C:\Users\reallylongusername\AppData\Local\Julia-0.4.0-pre`) may run into trouble.